config.go

```diff
@@ -462,6 +464,8 @@ type Config struct {
 }
 
 // DefaultConfig returns all default values for the Config struct.
+//
+// nolint:lll
```

Comment:
Does this mean that the `nolint` applies to all the go function?

if so, this is super cool :tada:

Reply:
yeah it applies to the whole function block